src/electron/platform/android/setup/steps/prompt-locate-adb.ts

```diff
@@ -8,7 +8,7 @@ export const promptLocateAdb: AndroidSetupStepConfig = deps => {
         actions: {
             saveAdbPath: (path: string) => {
                 deps.setAdbPath(path);
-                deps.stepTransition('prompt-connect-to-device');
+                deps.stepTransition('detect-adb');
```

Comment:
The steps *must* successfully go through the detect-adb step before progressing to anything that needs ADB